example-apps/github/test/authentication.test.js

```diff
@@ -51,3 +52,52 @@ describe('oauth2 app', () => {
     );
   });
 });
+
+describe('getAccessToken', () => {
+  let result;
+
+  beforeEach(async () => {
+    nock('https://github.com/login/oauth')
+      .post('/access_token')
+      .reply(200, { access_token: 'someAccessToken' });
+    result = await appTester(App.authentication.oauth2Config.getAccessToken);
```

Comment:
Nitpick: This result object seems to be part of the it('returns the expected tokens') test scope instead of the describe scope

```
    result = await appTester(App.authentication.oauth2Config.getAccessToken);
    expect(result.access_token).toBe('someAccessToken');
```